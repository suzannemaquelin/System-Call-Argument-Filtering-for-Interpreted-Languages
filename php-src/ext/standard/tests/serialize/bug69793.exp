Notice: Undefined property: Exception::$file in %s%ebug69793.php on line %d

Notice: Undefined property: Exception::$previous in %s%ebug69793.php on line %d

Notice: Undefined property: Exception::$previous in %s%ebug69793.php on line %d
string(41) "Exception in :1337
Stack trace:
#0 {main}"