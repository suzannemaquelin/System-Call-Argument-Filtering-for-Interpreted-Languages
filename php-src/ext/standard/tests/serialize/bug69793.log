
---- EXPECTED OUTPUT
Notice: Undefined property: Exception::$file in %s%ebug69793.php on line %d

Notice: Undefined property: Exception::$previous in %s%ebug69793.php on line %d

Notice: Undefined property: Exception::$previous in %s%ebug69793.php on line %d
string(41) "Exception in :1337
Stack trace:
#0 {main}"
---- ACTUAL OUTPUT
Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory

Notice: Undefined property: Exception::$file in /root/php-src/ext/standard/tests/serialize/bug69793.php on line 4

Notice: Undefined property: Exception::$previous in /root/php-src/ext/standard/tests/serialize/bug69793.php on line 4

Notice: Undefined property: Exception::$previous in /root/php-src/ext/standard/tests/serialize/bug69793.php on line 4
string(41) "Exception in :1337
Stack trace:
#0 {main}"
---- FAILED
