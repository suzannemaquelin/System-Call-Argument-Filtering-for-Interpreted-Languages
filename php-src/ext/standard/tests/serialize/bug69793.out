Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory

Notice: Undefined property: Exception::$file in /root/php-src/ext/standard/tests/serialize/bug69793.php on line 4

Notice: Undefined property: Exception::$previous in /root/php-src/ext/standard/tests/serialize/bug69793.php on line 4

Notice: Undefined property: Exception::$previous in /root/php-src/ext/standard/tests/serialize/bug69793.php on line 4
string(41) "Exception in :1337
Stack trace:
#0 {main}"